### PR TITLE
store (ticdc): refine create ti store

### LINKED ISSUE #5819: Make cdc can create mutiple tikv storage.
### Is your feature request related to a problem?

as title.

### Describe the feature you'd like

as title.

### Describe alternatives you've considered

_No response_

### Teachability, Documentation, Adoption, Migration Strategy

_No response_

### PR DESCRIPTION
**Pull request by asddongmen**
### What problem does this PR solve?

Issue Number: close #5819

#### Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

### Release note 

```release-note
Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

If you don't think this PR needs a release note then fill it with `None`.
```

### CODE REVIEW

**Review by CharlesCheung96 — APPROVED** [review]

File: cdc/kv/store_op.go [hunk 1]
```diff
@@ -34,26 +33,24 @@ func GetSnapshotMeta(tiStore tidbkv.Storage, ts uint64) (*meta.Meta, error) {
...
-
 	tiPath := fmt.Sprintf("tikv://%s?disableGC=true", urlv.HostString())
-	tiStore, err := store.New(tiPath)
+	securityCfg := tikvconfig.Security{
+		ClusterSSLCA:    credential.CAPath,
+		ClusterSSLCert:  credential.CertPath,
+		ClusterSSLKey:   credential.KeyPath,
+		ClusterVerifyCN: credential.CertAllowedCN,
+	}
+	d := driver.TiKVDriver{}
```
Comment (nongfushanquan) [inline cdc/kv/store_op.go:52]: could you add some comments about the advantage  of this refactor

Comment (asddongmen) [reply cdc/kv/store_op.go:52]: sure

Comment (asddongmen) [reply cdc/kv/store_op.go:52]: done

**Review by sdojjy — APPROVED** [review]

**Pull request merged**